topologies/datacenter/labguides/source/cvx.rst

```diff
@@ -47,7 +47,7 @@ CVX
             interface Ethernet4
               description HOST2
               channel-group 4 mode active
-              lacp rate fast
+              lacp timer fast
```

Comment:
This topo used the old version of EOS, so probably not change the CLI syntax on this topo:
```suggestion
              lacp rate fast
```